tools/cli/wskaction.py

```diff
@@ -170,7 +174,10 @@ def getExec(self, args, props):
             exe = self.getActionExec(args, props, pipeAction)
         elif args.artifact is not None and os.path.isfile(args.artifact):
             contents = open(args.artifact, 'rb').read()
-            if args.artifact.endswith('.swift'):
+            if args.kind in ['swift:3','swift:3.0','swift:3.0.0']:
```

Comment:
Why does the CLI expect the `runtime:version` syntax but this isn't reflected in the JSON object? I understand that later in code we can't have a class called e.g. `Swift:3Action`, but I would think at least the format for `--kind` and `kind` should match.